src/content/self-hosted/on-premise.mdx

```diff
@@ -0,0 +1,195 @@
+
+### Installing Helm v3
+
+Follow the official documentation for [installing the latest release of Helm v3](https://helm.sh/docs/intro/install/).
+
+### Creating the OAuth application
+
+> Okteto supports OAuth providers like Google, GitHub, or OpenID Connect to handle user authentication. For this guide, we'll focus on Okta with OpenID Connect. All the supported configuration settings [are described here](/docs/self-hosted/administration/configuration/#auth).
+
+Use this group of settings when using an OpenID Connect provider as your authentication provider.
+
+```yaml
+auth:
+  openid:
+    enabled: true
+    clientId: clientid
+    clientSecret: clientSecret
+    group: my-group
+    endpoints:
+      issuer: https://your-provider
+      authorization: https://your-provider/authorization
+    mapping:
+      externalIDKey: nickname
+      nameKey: name
+      emailKey: email
+      pictureKey: picture
+      groupsKey: groups
+```
+
+The `group` field is optional. Only members of the group will be allowed to log in into your Okteto instance. An empty `group` field permits any user to log in.
+
+The `issuer` and `authorization` endpoints must match the value returned in the provider config discovery.
+
+The `mapping` fields are optional. Use them to configure the mapping between Okteto's user attributes and the claim coming from your authentication provider.
+
+> Your provider needs to support the [UserInfo endpoint](https://openid.net/specs/openid-connect-core-1_0.html#UserInfo) in order to be used with Okteto. This authentication option follows the OpenID standard, and it has been validated with Okta, PingIdentity, and GitLab.
+
+## Adding the Okteto Helm repository
+
+You'll need to add the Okteto repository in order to be able to install Okteto:
+
+```console
+helm repo add okteto https://charts.okteto.com
+helm repo update
+```
+
+### Getting your Okteto License
+
+Okteto is free for small teams. You get all the features of [Okteto](/free-trial/) for up to 3 users with 3 namespaces each.
+
+Want to use Okteto with a bigger team? [Let's talk](https://okteto.com/#talktous)
+
+## Installing Okteto
+
+### Create the Okteto Namespace
+
+Run the following command to create a namespace to install Okteto in:
+
+```console
+$ kubectl create namespace okteto
+```
+
+### Configuring your Okteto instance
+
+Configure the Okteto helm values file with the following information:
+
+```yaml
+auth:
+  openid:
+    enabled: true
+    clientId: clientid
+    clientSecret: clientSecret
+    group: my-group
+    endpoints:
+      issuer: https://your-provider
+      authorization: https://your-provider/authorization
+    mapping:
+      externalIDKey: nickname
+      nameKey: name
+      emailKey: email
+      pictureKey: picture
+      groupsKey: groups
+
+email: admin@example.com
+license: 1234567890ABCD==
+subdomain: dev.example.com
+cluster:
+  endpoint: "https://your-kubernets-cluster-api-endpoint"
+
+ingress-nginx:
```

Comment:
I don't think this part fits the guide. Nginx Ingress can be installed on-premises.